codebuild/spec/buildspec_ktls.yml

```diff
@@ -0,0 +1,24 @@
+---
+version: 0.2
+batch:
+  build-list:
+    - identifier: UnitQEMUkTLS
+      debug-session: true
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+        image: aws/codebuild/standard:7.0
+        privileged-mode: true
+phases:
+  install:
+    commands:
+      - apt update
+      - apt upgrade -y
+      - apt install -y qemu qemu-system-x86 wget cloud-image-utils
+      - aws s3 --quiet sync s3://s2n-ktls-testing ./qemu
+
+  build:
+    commands:
+      - cd qemu; bash ./run.sh; cd ..
+      - rsync -avz --exclude=qemu -e 'ssh -p 2222' . codebuild@localhost:/home/codebuild/s2n-tls
+      - codebuild-breakpoint
+      - ssh -p 2222 codebuild@localhost "cd s2n-tls; sudo modprobe tls; nix develop .#openssl111 --command bash -c 'source ./nix/shell.sh && clean && configure && build && unit inet_socket_test'"
```

Comment:
Let's build all the tests. The tests are not the expensive part of this.

Unless some are failing in the environment?

Reply:
I've updated the buildspec todo all the tests.

Reply:
Are you sure? I still see "unit inet_socket_test"